src/eventBus.js

```diff
@@ -14,8 +14,7 @@
             return split && TestTrack._crx.persistAssignment(split.name, split.variant, 'crx', sharedSecret);
         });
 
-        // TODO: find a way to remove usage of jQuery
-        $.when.apply($, promises).done(function() {
+        Promise.allSettled(promises).then(function() {
```

Comment:
`TestTrack._crx.persistAssignment` will return a promise from axios, so now using `Promise.allSettled` should wait until all the requests complete before reloading the page